homeassistant/components/modbus/modbus.py

```diff
@@ -255,6 +255,10 @@ class ModbusHub:
     def __init__(self, hass: HomeAssistant, client_config: dict[str, Any]) -> None:
         """Initialize the Modbus hub."""
 
+        if CONF_CLOSE_COMM_ON_ERROR in client_config:
+            _LOGGER.warning(
```

Comment:
Why only log a warning? Why don't we create a repair issue?

Reply:
Because I don't think we can repair configuration.yaml, and it might also call for other parameters to be at least controlled.

Reply:
Sorry I meant issue. It will nudge the user they they should take action

Reply:
All the repair issues, I have seen so far, are config flow repairs, this integration do not use config flow it is solely configuration.yaml based.

But if you can point me in a direction of how to create a non config flow repair issue, I am happy to look at it.

Reply:
The repair component only have call for config flow, as far as I can see.

Reply:
Could probably use `cv.deprecated` in the schema instead.

Raising issues doesn't require a config flow.
```
async_create_issue(
            hass,
            DOMAIN,
            "deprecated_close_comm_config",
            breaks_in_ha_version="2024.4.0",
            is_fixable=False,
            severity=IssueSeverity.WARNING,
            translation_key="deprecated_close_comm_config",
        )
```
And then some added line in `string.json` and you have an issue created for the user which he can only resolve by removing the config element and restart HA.

Reply:
Thanks, will have a go of that.